src/extension/kernel/kernel.ts

```diff
@@ -1,175 +0,0 @@
-// import { format } from 'util';
-// import {
-//     notebook,
-//     NotebookCell,
-//     NotebookCellData,
-//     NotebookCellKind,
-//     NotebookCellMetadata,
-//     NotebookCellOutput,
-//     NotebookCellOutputItem,
-//     NotebookRange,
-//     NotebookDocument,
-//     TextEditor,
-//     workspace,
-//     WorkspaceEdit,
-//     NotebookKernelPreload
-// } from 'vscode';
-// import { Client } from '../kusto/client';
-// import { getChartType } from '../output/chart';
-// import { createPromiseFromToken } from '../utils';
-// import { isKustoInteractive } from './provider';
-
-// export class Kernel implements NotebookKernel {
-//     // todo@API make this mandatory?
-//     public readonly id = 'kusto';
-
-//     public readonly label = 'Kusto';
-//     public readonly description = 'Execute Kusto Queries';
-//     public readonly detail = '';
-//     public readonly isPreferred: boolean = true;
-//     public readonly preloads: NotebookKernelPreload[] = [];
-//     public readonly supportedLanguages?: string[] = ['kusto'];
-//     private static interactiveKernel?: Kernel;
-//     public static get InteractiveKernel() {
-//         return Kernel.interactiveKernel;
-//     }
-//     constructor(public readonly document: NotebookDocument) {
-//         if (isKustoInteractive(document)) {
-//             Kernel.interactiveKernel = this;
-//         }
-//     }
-//     public async executeInteractiveSelection(textEditor: TextEditor): Promise<void> {
```

Comment:
The old hacky way of displaying a raedonly Notebook can now go away.... (my previous solution was pretty much the same)